fapolicy_analyzer/ui/trust_file_list.py

```diff
@@ -160,7 +173,8 @@ def process_rows(queue, total):
                 return False
 
             count = self._get_tree_count()
-            if count < total:
+
+            if count < self.total:
```

Comment:
Use `total` here not `self.total` since you are in a different thread here. Probably safe since you are only reading it, but you have the local version availble so might as well use it.

Reply:
I need to read from the other thread in order to get the final count of files with discrepancies or else we never meet the end conditions to exit and we only get the final count once everything is loaded.

Reply:
Ok, I see your issue there. I don't think this approach of setting the new `total` value from the `system_trust_database_admin` on load completion is the best. On my system the percent complete in the list status bar moves very slowly up to about 22% then jumps right to completed. 

I'm thinking you'll need to update the `total` in the `append_trust` method. Update it by subtracting all the entries that where not processed because they were trusted from the total every run of the `process_trust` thread (watch thread safety here). That wouldn't be perfect but it would be better. At least the total number would slowly decrease over time instead of all at once at the end. Thoughts?